firedrake/plot.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+import numpy as np
+from ufl import cell
+from firedrake import Function, SpatialCoordinate
+
+
+def plot_mult(functions, num_points=100):
+    try:
+        import matplotlib.pyplot as plt
+        from matplotlib.widgets import Slider
+    except ImportError:
+        raise RuntimeError("Matplotlib not importable, is it installed?")
+    figure, ax = plt.subplots()
+    func_axis = plt.axes([0.25, 0.025, 0.65, 0.03], axisbg='lightgoldenrodyellow')
+    func_slider = Slider(func_axis, "Func Select", 0.1, len(functions), valinit=0)
+    func_slider.valtext.set_text('0')
+
+    def _update(val):
+        val = int(val - 0.1)
+        func_slider.valtext.set_text('{:.0f}'.format(val))
+        ax.cla()
+        plot(functions[val], ax)
+    _update(0)
+    func_slider.on_changed(_update)
+    return figure
+
+
+def plot(function, axes=None, num_points=100, **kwargs):
+    """Plot a function and return a matplotlib figure object.
+    :arg function: The function to plot.
+    :arg num_points: Number of points per element
+    :arg kwargs: Additional keyword arguments passed to
+    ``matplotlib.plot``.
+    """
+    try:
+        import matplotlib.pyplot as plt
+    except ImportError:
+        raise RuntimeError("Matplotlib not importable, is it installed?")
+    if function.function_space().mesh().ufl_cell() == cell.Cell("interval"):
+        if function.function_space().ufl_element().degree() < 4:
+            return bezier_plot(function, axes)
+        points = one_dimension_plot(function, num_points)
+    else:
+        raise RuntimeError("Unsupported functionality")
+    if axes is None:
+        axes = plt.subplot(111)
+    axes.plot(points[0], points[1], **kwargs)
+    return plt.gcf()
+
+
+def one_dimension_plot(function, num_points):
```

Comment:
Can you rename this function, given that it doesn't actually plot anything to `compute_points`?